src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java

```diff
@@ -183,216 +169,179 @@ public class ProGuardMojo extends AbstractMojo {
-	 *
-	 * @parameter default-value="small"
 	 */
+	@Parameter(defaultValue = "small")
 	private String attachArtifactClassifier;
 
 	/**
 	 * Whether to append the {@link #attachArtifactClassifier} to the artifact final name.
-	 *
-	 * @parameter default-value="true"
 	 */
+	@Parameter(defaultValue = "true")
 	private boolean appendClassifier;
 
 	/**
 	 * Whether to include {@code META-INF/MANIFEST.MF} file
-	 *
-	 * @parameter default-value="false"
 	 */
+	@Parameter(defaultValue = "false")
 	private boolean addManifest;
 
 	/**
 	 * Whether to include {@code META-INF/maven/**} Maven descriptor.
-	 *
-	 * @parameter default-value="false"
 	 */
+	@Parameter(defaultValue = "false")
 	private boolean addMavenDescriptor;
 
 	/**
 	 * Directory containing the input and generated JAR.
-	 *
-	 * @parameter property="project.build.directory"
-	 * @required
 	 */
+	@Parameter(required = true, defaultValue = "${project.build.directory}")
 	protected File outputDirectory;
 
 	/**
 	 * The Maven project reference where the plugin is currently being executed. The default value is populated from
 	 * Maven.
-	 *
-	 * @parameter property="project"
-	 * @readonly
-	 * @required
 	 */
+	@Parameter(readonly = true, required = true, defaultValue = "${project}")
 	protected MavenProject mavenProject;
 
 	/**
 	 * The plugin dependencies.
-	 *
-	 * @parameter property="plugin.artifacts"
-	 * @required
-	 * @readonly
 	 */
+	@Parameter(readonly = true, required = true, defaultValue = "${plugin.artifacts}")
 	protected List<Artifact> pluginArtifacts;
 
-	/**
-	 * @component
-	 */
+	@Component
 	private MavenProjectHelper projectHelper;
 
 	/**
 	 * The Jar archiver.
-	 *
-	 * @component role="org.codehaus.plexus.archiver.Archiver" roleHint="jar"
-	 * @required
 	 */
+	@Component(role = Archiver.class, hint = "jar")
 	private JarArchiver jarArchiver;
 
 	/**
 	 * The Maven archive configuration to use. Only if {@link #assembly} is used.
-	 *
-	 * @parameter
 	 */
+	@Parameter
 	protected MavenArchiveConfiguration archive = new MavenArchiveConfiguration();
 
 	/**
 	 * The max memory the forked Java process should use, e.g. 256m
-	 *
-	 * @parameter
 	 */
+	@Parameter
 	protected String maxMemory;
 
 	/**
 	 * ProGuard main class name.
-	 *
-	 * @parameter default-value="proguard.ProGuard"
 	 */
-	protected String proguardMainClass = "proguard.ProGuard";
+	@Parameter(defaultValue = "proguard.ProGuard")
+	protected String proguardMainClass;
 
 	/**
 	 * Sets the name of the ProGuard mapping file.
-	 *
-	 * @parameter default-value="proguard_map.txt"
 	 */
-	protected String mappingFileName = "proguard_map.txt";
+	@Parameter(defaultValue = "proguard_map.txt")
+	protected String mappingFileName;
 
 	/**
 	 * Sets the name of the ProGuard seed file.
-	 *
-	 * @parameter default-value="proguard_seed.txt"
 	 */
-	protected String seedFileName = "proguard_seeds.txt";
+	@Parameter(defaultValue = "proguard_seed.txt")
```

Comment:
There was also a bug here previously where the initial field value had an extra `s` after `seed`: <code>proguard_seed<ins>s</ins>.txt</code>

But I think the `default-value` had higher precedence anway, and overwrote the initial field value.